src/lib/datepicker/datepicker.spec.ts

```diff
@@ -30,13 +30,17 @@ import {MatDatepickerToggle} from './datepicker-toggle';
 import {MAT_DATEPICKER_SCROLL_STRATEGY, MatDatepickerIntl, MatDatepickerModule} from './index';
 import {Subject} from 'rxjs';
 import {Directionality} from '@angular/cdk/bidi';
+import {BrowserDynamicTestingModule} from '@angular/platform-browser-dynamic/testing';
 
 describe('MatDatepicker', () => {
   const SUPPORTS_INTL = typeof Intl != 'undefined';
 
   // Creates a test component fixture.
-  function createComponent(component: any, imports: any[] = [], providers: any[] = []):
-    ComponentFixture<any> {
+  function createComponent(
+    component: any,
+    imports: any[] = [],
+    providers: any[] = [],
+    entryComponents: any[] = []): ComponentFixture<any> {
```

Comment:
nit: you could use better types instead of `any`:

```
import {Component, FactoryProvider, Type, ValueProvider, ViewChild} from '@angular/core';

...

function createComponent(
  component: Type<any>,
  imports: Type<any>[] = [],
  providers: (ValueProvider | FactoryProvider)[] = [],
  entryComponents: Type<any>[] = []): ComponentFixture<any> {
  ...
}
```

Reply:
done in https://github.com/angular/material2/pull/10856/commits/811ee17648d88af4b4d85ddf54de4ce913e0715f